packages/console/src/components/Card/index.module.scss

```diff
@@ -4,5 +4,5 @@
   background: var(--color-on-primary);
   border-radius: _.unit(4);
   padding: _.unit(6);
-  height: 100%;
+  min-height: 100%;
```

Comment:
This will cause a problem in the flex layout.
<img width="1264" alt="image" src="https://user-images.githubusercontent.com/10806653/160309186-9df6699d-8828-462c-830e-4c386683b3a6.png">

Reply:
Would you mind opening a PR to fix this? Thanks.